code-coverage.sh

```diff
@@ -1,3 +1,4 @@
+cd stellar_rust_sdk
```

Comment:
Dit kan, maar is niet zo fraai. Mooiste is om `cargo test` met een manifest file aan te roepen. Mocht je relatieve paden met dependencies hebben (zoals de derive crate) dan werkt dat ook mooi. Zie onderstaande suggestie.